posixlib/src/main/scala/scala/scalanative/posix/spawn.scala

```diff
@@ -0,0 +1,177 @@
+package scala.scalanative.posix
+
+import scalanative.unsafe._
+import scalanative.unsafe.Nat._
+
+import scala.scalanative.posix.sys.types
+
+/** POSIX spawn.h for Scala
+ *
+ *  The Open Group Base Specifications
+ *  [[https://pubs.opengroup.org/onlinepubs/9699919799 Issue 7, 2018]] edition.
+ *
+ *  A method with a PS comment indicates it is defined in POSIX extension
+ *  "Process Scheduling", not base POSIX.
+ */
+@extern
+object spawn {
+
+  /* posix_spawnattr_t & posix_spawn_file_actions_t are opaque bulk storage
+   * types. They have no user accessible fields, not even the component Bytes.
+   * Users of these types should leave them opaque: i.e. no read, no write.
+   *
+   * The sizes here are from Linux 6.0.  Code in spawn.c checks at compile-time
+   * that these sizes are greater than or equal to the equivalent OS types.
+   * Maintainers:  If you change sizes, either here or in spawn.c, change
+   *               the other file also.
+   */
```

Comment:
@ekrich Thank you for the review & the suggestion.

I have let your suggestion brew on my back brain for several hours.  I come
back to my original thoughts:  These are developer focused comments and
should not be Scaladoc. 

I am a great believer in doc starting with a best guess "appropriate precision" and
supplying easy ways to get both less (dictionaries & tutorials) or more (read the source)
information.  Finding that sweet spot is art and sometimes skill.

We both have struggled, or at least I have, with reading and interpreting the Open Group POSIX
specification.  Sometimes implementation depends on a particular reading of the specification.
The first paragraph is stating my understanding of what the specification means when it does
not give structure members or layout.  In other places these are referred to as 'opaque'.
I was trying to give _developers_ a hint beyond a mutable `Array[Byte]` why they were not
seeing a corresponding `struct` declaration.   This implementation/interpretation detail should
not be hidden from an end user: they can read the source.  Since it does not affect their usage
of types described by POSIX, it is has negative value (i.e. source of confusion) at the Scaladoc level.

Certainly the second paragraph is "developer only". It is intended to teach cross ties to 
"new" or returning-after-a-long-hiatus maintainers. 

Based on your being an independent set of eyes looking at the comment, can you suggest
alternate Scaladoc wording for the first comment?